traveler-ios-ui/TravelerKitUI/Storyboards/Product.storyboard

```diff
@@ -3,7 +3,7 @@
     <device id="retina6_1" orientation="portrait" appearance="light"/>
     <dependencies>
         <deployment identifier="iOS"/>
-        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="15509"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="15510"/>
```

Comment:
Same?

Reply:
It's because of the segue change I think.

Reply:
It's because of the segue change I think.